clang/lib/DPCT/APINames.inc

```diff
@@ -1445,14 +1445,128 @@ ENTRY(cuStreamWriteValue64, cuStreamWriteValue64_v2, false, NO_FLAG, P4, "commen
 ENTRY(cuFuncSetAttribute, cuFuncSetAttribute, false, NO_FLAG, P4, "comment")
 ENTRY(cuLaunchCooperativeKernel, cuLaunchCooperativeKernel, false, NO_FLAG, P4, "comment")
 ENTRY(cuLaunchCooperativeKernelMultiDevice, cuLaunchCooperativeKernelMultiDevice, false, NO_FLAG, P4, "comment")
-ENTRY(this_thread_block, this_thread_block, true, NO_FLAG, P4, "Successful")
-ENTRY(this_multi_grid, this_multi_grid, false, NO_FLAG, P4, "comment")
-ENTRY(tiled_partition, tiled_partition, true, NO_FLAG, P4, "If size equals to 32, tool will migrate it, else warning is emitted.")
-ENTRY(thread_block_tile, thread_block_tile, false, NO_FLAG, P4, "comment")
-ENTRY(coalesced_threads, coalesced_threads, false, NO_FLAG, P4, "comment")
-ENTRY(coalesced_group, coalesced_group, false, NO_FLAG, P4, "comment")
-ENTRY(grid_group, grid_group, false, NO_FLAG, P4, "comment")
-ENTRY(thread_rank, thread_rank, true, NO_FLAG, P4, "Successful")
+
+// Coopeartive_group migration.
+ENTRY(cooperative_groups::__v1::thread_rank, cooperative_groups::__v1::thread_rank, true, NO_FLAG, P4, "Successful")
```

Comment:
__v1,  as it is enabled by default;  let's remove it to make it by default work for v2.

Reply:
We need the __v1 to match the AST. If the v2 will be enabled in the future will need to be added in the future.